### ISSUE TITLE
Documentation Should at Least Give a *Hint* on How to Export HTML

### ISSUE THREAD

**Issue by machineghost** [facebook/draft-js#2564]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
Draft emphatically refuses to tell you how to do the most basic thing (get HTML out of the editor).

**What is the expected behavior?**

The library should tell you how to get HTML out.

Look, I get that you guys have some crazy philosophical objection to adding a `toHTML` method to `contentState`.  I get it ... I mean, I don't, *at all*, but you're the maintainers and you know way more than I do, and I'm sure you have perfectly good reasons (again, for your strong opposition to giving users what they want).

But for Pete's sake, can't you at least point users in the right direction in the docs?

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
Current

**Comment by NaviMarella** [comment]
I agree with @machineghost, We should add have an `ecosystem`  section for `draft js` related libraries added in docs.

Here is an example which redux had and useful for library users, https://redux.js.org/introduction/ecosystem

**Comment by mrkev** [comment]
Sounds like a good idea. Happy to review any PRs related to this (:
